pkg/test/framework/components/echo/config.go

```diff
@@ -138,6 +138,12 @@ type Config struct {
 	// the CUSTOM authorization policy when the ext-authz server is deployed locally with the application container in
 	// the same pod.
 	IncludeExtAuthz bool
+
+	// IPFamily for the service
```

Comment:
It should be clarified what happens if the user does not set them to the echo service or that this should only be changed when doing dual-stack testing.